romancal/tweakreg/tweakreg_step.py

```diff
@@ -6,35 +6,16 @@
 from pathlib import Path
 
 import numpy as np
-from astropy import units as u
-from astropy.coordinates import SkyCoord
 from astropy.table import Table
 from roman_datamodels import datamodels as rdm
-from tweakwcs.correctors import JWSTWCSCorrector
-from tweakwcs.imalign import align_wcs
-from tweakwcs.matchutils import XYXYMatch
+from stcal.tweakreg import tweakreg
 
 # LOCAL
 from ..datamodels import ModelLibrary
 from ..stpipe import RomanStep
-from . import astrometric_utils as amutils
 
-
-def _oxford_or_str_join(str_list):
-    nelem = len(str_list)
-    if not nelem:
-        return "N/A"
-    str_list = list(map(repr, str_list))
-    if nelem == 1:
-        return str_list
-    elif nelem == 2:
-        return f"{str_list[0]} or {str_list[1]}"
-    else:
-        return ", ".join(map(repr, str_list[:-1])) + ", or " + repr(str_list[-1])
-
-
-SINGLE_GROUP_REFCAT = ["GAIADR3", "GAIADR2", "GAIADR1"]
-_SINGLE_GROUP_REFCAT_STR = _oxford_or_str_join(SINGLE_GROUP_REFCAT)
+SINGLE_GROUP_REFCAT = tweakreg.SINGLE_GROUP_REFCAT
+_SINGLE_GROUP_REFCAT_STR = tweakreg._SINGLE_GROUP_REFCAT_STR
```

Comment:
```suggestion
```
I added a suggestion above to import these directly from stcal instead of re-assigning them to variables here.